src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-packageid.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Id of the package in xl-deploy which needs to be deployed. Jenkin's environment variables can be used to specify this value.
```

Comment:
ID of the package in XL Deploy that should be deployed. Jenkins environment variables can be used to specify this value.